### DISCUSSION TITLE
Add grouping for repositories.

### DISCUSSION THREAD

**Discussion by herboren** [community/community#42083] [Repositories]
### Select Topic Area

Product Feedback

### Body

This is was initially addressed in 2021, ID: 5759, when someone requested grouping for repositories for a more structed layout on projects. I generally enjoy modding for multiple games and some of my own personal projects which fall into another type of categorie. However, due to GitHub lack of features, I store these projects in separate folders on my desktop. I am beginning to create an endless list on my repository page, and this is where structure comes into play. It would be nice to group projects into Categories to reduce clutter in the profile page.

**Comment by TruDan** [comment]
+1 to this feature! It's one thing I really miss from GitLab :( Our company has several departments using GitHub. Back in the day when we used GitLab instead (switched to GitHub since GitLab was having a lot of downtime and just became unreliable - do believe its better now though) we would have different folders for our repos

Like:
/Company/Hardware/Device 1 Firmware.git
/Company/Cloud/Backend/Some API.git
/Company/Cloud/Frontend/Mobile App 1.git
/Company/Cloud/Frontend/Web App.git
etc..

This was really nice for finding projects from the top-level company page.
I believe you could also assign permissions on folder level too - although this we already do with a tool we made which assigns permissions based on what we tagged the repo's with, and for now we just use the tags for filtering.

**Reply by merusso** [reply]
@TruDan, I assume you're referring to GitLab's [Groups](https://docs.gitlab.com/ee/user/group/#groups) and [Subgroups](https://docs.gitlab.com/ee/user/group/subgroups/index.html) feature. It would be nice to have some sort of repository grouping like this.

Repository "topics" are a form of grouping -- arguably more flexible since repos can belong to multiple topics -- but there doesn't seem to be a way (in the UI) to list the available topics for an Organization. I see a feature request here: https://github.com/orgs/community/discussions/44672

**Comment by cschar** [comment]
+1 to this, I have so many private repositories all on one github account. It would be nice to have better organization of these repos.
